tools/build_defs/repo/jvm.bzl

```diff
@@ -150,8 +151,13 @@ def _serialize_given_rule_import(rule_name, name, path, srcpath, attrs, props, a
     lines = [
         "%s(" % rule_name,
         "    name = %s," % repr(name),
-        "    jars = [%s]," % repr(path),
     ]
+
+    if rule_name == "aar_import":
+        lines.append("    aar = %s," % repr(path))
```

Comment:
Can we parameterize this as well? Maybe get a dict with the packaging inside as well

Reply:
I'm not sure how to do this elegantly without exposing the internals of the repository rule.

If we parameterize this attribute, we'll need to pass in something like `rule_import_attribute = "jars = [%s]"` or `rule_import_attribute = "aar = %s"`.